docs/jwt-ory-to-istio.md

```diff
@@ -136,3 +136,17 @@ Istio doesn't support regex type of path matching in Authorization Policies, whi
 Istio doesn't support configuring a JWT token from `cookie`, and Ory Oathkeeper does. Istio supports only `fromHeaders` and `fromParams` configurations.
 
 Using Istio as JWT access strategy requires the workload behind the service to be in the service mesh, for example, to have the Istio proxy. To learn how to add workloads to the Istio service mesh read the [Istio documentation](https://istio.io/latest/docs/ops/common-problems/injection/).
+
+## Change of status `401` to `403` when calling an endpoint without `Authorization` header
+
+With the introduction of Istio JWT the calls without a token will now fail on `AuthorizationPolicy` that forces `requestPrincipals` to not be empty. This results in a behaviour of requests without an `Authorization` header to fail on `AuthorizationPolicy` validation step, instead of `RequestAuthentication` that allows tokens without the header. In conclusion this behaviour will make the request fail with code `403` instead of `401`, which was the case for Oathkeeper JWT.
```

Comment:
i think this sentence should be more user friendly. Start with explanation on behaviour change avoiding technical details. Users might not be aware of what is created underneath APIRule. Something like: In case of ORY Oathkeeper a call without JWT to the secured workload resulted in `401`. Istio based JWT changes that behaviours and the call without the token results in `403`. + link to Istio docu.

cc: @nataliasitko

Reply:
```suggestion
Previously, when using ORY Oathkeeper, if you called a secured workload without a JWT token, it resulted in the `401` error. This behavior has changed with the implementation of Istio-based JWT. Now, calls made without a token result in the `403` error. To learn more, read the Istio documentation on [RequestAuthentication](https://istio.io/latest/docs/concepts/security/#request-authentication) and [AuthorizationPolicy](https://istio.io/latest/docs/reference/config/security/authorization-policy).
```
Would that be okay?